Samples/iOS-Swift/iOS-Swift.xcodeproj/project.pbxproj

```diff
@@ -1378,7 +1378,7 @@
 				MARKETING_VERSION = 7.27.0;
 				PRODUCT_BUNDLE_IDENTIFIER = "io.sentry.sample.iOS-Swift";
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AppStore io.sentry.sample.iOS-Swift";
+				PROVISIONING_PROFILE_SPECIFIER = "match Development io.sentry.sample.iOS-Swift";
```

Comment:
Yes, because I needed the sample app to build under the TEST configuration, and the signing settings were all mixed up for that config. So that was causing saucelabs build failures, see this earlier build before I fixed the codesign settings: https://github.com/getsentry/sentry-cocoa/actions/runs/4496464429/jobs/7911150807?pr=2823#step:9:366

We had a mix of distribution/development signing settings in the TEST configs (screenshot) and [specified "development" for the fastlane build](https://github.com/getsentry/sentry-cocoa/blob/725565ade20633f9aa726f67daf1a46315c8cfbf/fastlane/Fastfile#L108), which I believe would be the correct signing setting for a CI test run, vs pulling in the production key material.

The mismatch between the distribution identity setting and development parameter in the fastfile is the mismatch that actually caused the failure.

![Screenshot 2023-03-24 at 11 10 27 AM](https://user-images.githubusercontent.com/3241469/227619316-947a7891-3754-4199-a8b4-5c14377dfa23.jpg)

Reply:
Missed the settings for iOS-SwiftClip, which is what was causing the failure you linked. Fixed in https://github.com/getsentry/sentry-cocoa/pull/2823/commits/3827deef71debe5749821721685cec34349f4d1f

Reply:
Thanks for the clarification.